PayKids/presentation/src/main/java/com/paykids/presentation/view/mypage/MyPageFragment.kt

```
@@ -0,0 +1,78 @@
+import com.paykids.presentation.databinding.FragmentMypageBinding
+import com.paykids.presentation.utils.UiState
+import com.paykids.presentation.view.home.HomeActivity
+import com.paykids.presentation.view.signIn.SignActivity
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class MyPageFragment : BaseFragment<FragmentMypageBinding>() {
+    private val homeViewModel: MyPageViewModel by viewModels()
+
+    override fun initView() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)